api/fixtures/example.go

```diff
@@ -271,6 +272,30 @@ func (o ExampleOptions) Resources() *ExampleResources {
 		default:
 			panic(fmt.Sprintf("service publishing type %s is not supported", o.Kubevirt.ServicePublishingStrategy))
 		}
+		if o.Kubevirt.ExtClusterKubeConfig != nil {
+			extKubeConfigSecret := &corev1.Secret{
+				TypeMeta: metav1.TypeMeta{
+					Kind:       "Secret",
+					APIVersion: corev1.SchemeGroupVersion.String(),
+				},
+				ObjectMeta: metav1.ObjectMeta{
+					Name:      o.Name + "-infra-credentials",
+					Namespace: o.Namespace,
+				},
+				Data: map[string][]byte{
+					"kubeconfig": []byte(base64.StdEncoding.EncodeToString(o.Kubevirt.ExtClusterKubeConfig)),
```

Comment:
@orenc1 Oh, maybe this is the issue. Upstream, we didn't base64 encode the kubeconfig when this is being tested.

https://github.com/kubernetes-sigs/cluster-api-provider-kubevirt/blob/884c8ca19f57c6567a8661ab244344f9b587648e/e2e/create-cluster_test.go#L908

Reply:
yes, that was the issue. thanks.